suse_migration_services/migration_config.py

```diff
@@ -42,11 +46,13 @@ def get_migration_product(self):
 
         The value returned is passed to the --product option in the
         zypper migration call
-
-        migration:
-          target:
-            product: product_name
         """
-        migration = self.config_data.get('migration')
-        if migration and migration.get('target'):
-            return migration.get('target').get('product')
+        migration_product = self.config_data.get('migration_product')
+        if not migration_product:
+            message = (
+                'Migration product not found, aborting migration attempt.'
+            )
+            log.error(message)
```

Comment:
why use a message variable for static text ?

```python
log.error(
    'Migration product not found, aborting migration attempt.'
)
```

Reply:
To re use it in the exception

Reply:
@schaefi if you are ok with the message variable to be re-used in the exception, I will rebase the 7 commits (2b11417 to fb21d4d) into 1 and merge.

Reply:
Oh I see, sorry Jesus, overlooked that part. Sure that's all fine then